be/src/column/datum_row.h

```diff
@@ -0,0 +1,131 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#pragma once
+
+#include "column/row.h"
+#include "util/int96.h"
+
+namespace starrocks {
+
+// An implementation of row backed with a vector of datum.
+class DatumRow final : public Row {
```

Comment:
What's the different from DatumTuple?

Reply:
1. The size of `DatumRow` is fixed, and can update any field by the ordinal so that the row can be reused
2. `DatumRow` will return the data with original type directly rather than a `Datum`, and I think it may be more easy to use
3. `DatumRow` is an implementation of the abstraction `Row`